devicemodel/hw/pci/passthrough.c

```diff
@@ -814,6 +814,7 @@ passthru_init(struct vmctx *ctx, struct pci_vdev *dev, char *opts)
 			break;
 		}
 	}
+	pci_iterator_destroy(iter);
```

Comment:
There are some **goto** statement used in the function for some error cases.
If you put pci_iterator_destroy(iter) here, some cases are still not being handled properly.